packages/@dcl/inspector/src/lib/sdk/operations/add-component.ts

```diff
@@ -0,0 +1,13 @@
+import { Entity, IEngine } from '@dcl/ecs'
+import { isLastWriteWinComponent } from '../../../hooks/sdk/useComponentValue'
+
+export function addComponent(engine: IEngine) {
+  return function addComponent(entity: Entity, componentId: number) {
+    const component = engine.getComponent(componentId)
+    if (isLastWriteWinComponent(component)) {
```

Comment:
should we thrown an error if you pass another component ? or maybe a log ?

Reply:
added a throw 👍